include/gz/rendering/base/BaseStorage.hh

```diff
@@ -854,7 +857,12 @@ namespace gz
     typename BaseStore<T, U>::ConstUIter
     BaseStore<T, U>::ConstIterByName(const std::string &_name) const
     {
-      return this->store.find(_name);
+      auto idx = this->storeMap.find(_name);
+      if (idx == this->storeMap.end())
+      {
+        return this->store.end();
```

Comment:
```suggestion
        return this->storeMap.end();
```

or `return idx;`

Reply:
I think it's suppose to be `this->store.end();` because the return type is `ConstUIter` which is typedef'd to `std::vector`.